### DISCUSSION TITLE
soy nuevo

### DISCUSSION THREAD

**Discussion by roggsenn** [sherlock-project/sherlock#2361] [Q&A]
soy nuevo usando github y no se como compilar el proyecto, si pudieran ayudarme seria de mucha ayuda, logre abrirlo en visual studio code pero de ahi no se que mas hacer:(
![Captura de pantalla 2024-11-09 224411](https://github.com/user-attachments/assets/44224ee8-1ace-4969-a5a0-2dfab385d425)

**Comment by bytexenon** [comment]
ES:
Pido disculpas por mi mal español, pero espero que esto sea claro:

No necesitas compilar el proyecto, y no necesitas descargar Sherlock desde la fuente a menos que quieras modificar alguno de sus archivos.

Para instalar Sherlock, usa el siguiente comando de `pipx`: `pipx install sherlock-project` o `pip install --user sherlock-project`. Después de la instalación, puedes ejecutarlo con: `sherlock tu_nombre_de_usuario`. Para obtener instrucciones detalladas de instalación, visita [este enlace](https://sherlockproject.xyz/installation) (nota: está en inglés).

Si deseas modificar alguno de los archivos del proyecto y ejecutar tu versión modificada, sigue estos pasos:

1. Clona el repositorio desde `https://github.com/sherlock-project/sherlock` o descárgalo como un archivo ZIP y descomprímelo.
2. Navega a la carpeta del proyecto en la terminal con `cd sherlock`. Si estás usando VSCode, selecciona "..." desde la barra superior, luego "Terminal" -> "Nuevo Terminal" para abrir una ventana de terminal en la carpeta.
3. Haz los cambios que desees en el proyecto.
4. Instálalo ejecutando `pipx install . --force` o `pip install .`
5. Ejecuta tu versión modificada de Sherlock usando el comando `sherlock`, de este modo: `sherlock tu_nombre_de_usuario_aquí`

EN:
I apologize for my bad Spanish, but I hope this is clear:

You don't need to compile the project, and you don't need to download Sherlock from source unless you want to modify any of its files. 

To install Sherlock, use the following `pipx` command: `pipx install sherlock-project` or `pip install --user sherlock-project`. After installation, you can run it with: `sherlock your_username`. For detailed installation instructions, visit [this link](https://sherlockproject.xyz/installation) (note: it's in English).

If you want to modify any of the project's files and run your modified version, follow these steps:

1. Clone the repository from `https://github.com/sherlock-project/sherlock` or download it as a ZIP file and unzip it.
2. Navigate to the project folder in the terminal with `cd sherlock`. If you're using VSCode, select "..." from the top bar, then "Terminal" -> "New Terminal" to open a terminal window in the folder.
3. Make your desired changes to the project.
4. Install it by running `pipx install . --force` or `pip install .`
5. Run your modified version of Sherlock using the `sherlock` command, like so: `sherlock your_username_here`
